front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/index.tsx

```diff
@@ -7,6 +7,7 @@ import useFeatureFlag from 'hooks/useFeatureFlag';
 import { Box } from '@citizenlab/cl2-component-library';
 import TextResponses from './TextResponses';
 import Analysis from './Analysis';
+import AnalysisUpsell from './AnalysisUpsell';
```

Comment:
We only include the AnalysisUpsell for native surveys? Not for ideation?

Reply:
Correct! :)